### PR TITLE
Certbot initial commits

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

File: bin/run_certbot.sh [hunk 1]
```diff
@@ -0,0 +1,193 @@
+#!/bin/sh
+# shellcheck disable=SC2039,SC2113
+
+set -e
+
+print_usage() {
+cat << EOF
+usage: ${0} options
+
+This script helps to update content views in Foreman.
```
Comment (mroote) [inline bin/run_certbot.sh]: Woops copy/paste error.

Comment (hrpatel) [reply bin/run_certbot.sh]: Doh.. I missed that too.. updating

Comment (hrpatel) [reply bin/run_certbot.sh]: Updated

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM certbot/certbot:v0.27.1
+
+LABEL name="certbot"
+LABEL version="latest"
+
+RUN apk update && \
```
Comment (adamgilbert) [inline Dockerfile]: I believe you can use  --no-cache to run apk update and manually removing the cache: https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

**Review by adamgilbert — DISMISSED** [review]
+1 if it works!

File: Dockerfile (hunk 2 above)
Comment (hrpatel) [reply Dockerfile]: Done

Comment (hrpatel) [reply Dockerfile]: Thanks for the tip

**Review by antevens — COMMENTED** [review]
How about baking this in as a separate script under the shtdlib repo?

**Comment by hrpatel** [comment]
@antevens Which ones? Certbot? Build? Both?

**Comment by antevens** [comment]
I was thinking the certbot stuff might be useful for others ....

**Comment by hrpatel** [comment]
Agreed, I've made an issue to track it: https://github.com/SecurityCompass/certbot-docker/issues/2

**Review by antevens — DISMISSED** [review]

**Comment by mroote** [comment]
Might be good to add some instructions in the readme about how to link the container volumes into a separate web server, or maybe an example config?

**Comment by hrpatel** [comment]
@mroote Done

**Review by mroote — APPROVED** [review]
Nice +1

**Pull request merged**